lib/src/themes/theme_reader.dart

```diff
@@ -203,11 +203,26 @@ class ThemeReader {
   }
 
   LineLayout _toLineLayout(jsonLayer) {
+    List<num> dashArray = [];
+
+    // check if we need to paint line dashed
+    final dashJson = jsonLayer['paint']['line-dasharray'];
```

Comment:
[line-dasharray is a paint property](https://docs.mapbox.com/mapbox-gl-js/style-spec/layers/#paint-line-line-dasharray) - this should be moved to `PaintFactory`

Reply:
Done, thank you!